{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -1,36 +1,53 @@
-language: python
-python: '3.5'
 sudo: false
+language: python
+python:
+{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
+  - "{{ env }}"
+{% endfor %}
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
   global:
-    LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS="abrt segv"
+{% endif %}
   matrix:
-    - TOXENV=check
-{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
-    - TOXENV={{ env }}-cover
+    - COVERAGE=cover
       {%- if cookiecutter.coveralls|lower == 'yes' %}
         {%- if cookiecutter.c_extension_support|lower == "yes" %},extension-coveralls{% endif %},coveralls
       {%- endif %}
       {%- if cookiecutter.codecov|lower == 'yes' %},codecov{% endif %}
-    - TOXENV={{ env }}-nocov
-{% endfor %}
+    - COVERAGE=nocov
+matrix:
+  include:
+    - python: "3.4"
+      env: TOXENV=check
+
 before_install:
-  - python --version
```

Comment:
What I am trying to say is that adding `python --version` just runs the same command one more time:

```
travis_cmd source\ \~/virtualenv/python2.7/bin/activate --assert --echo --timing
travis_cmd python\ --version --echo
travis_cmd pip\ --version --echo

travis_fold start before_install.1
  travis_cmd python\ --version --assert --echo --timing
travis_fold end before_install.1
```

And we get the obvious [repetition](https://travis-ci.org/abusalimov/test/jobs/90952855#L92-L97):
```
$ python --version
Python 2.7.9
$ pip --version
pip 6.0.7 from /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages (python 2.7)
$ python --version
Python 2.7.9
```

I won't insist on this change though. Would you like me to rollback the whole a5d525fe8cc15739fda085e7be217d94ae450874 ("travis: Don't show --version for tools already shown by Travis")?